microsoft-edge/webdriver-chromium/index.md

```diff
@@ -372,6 +372,17 @@ For more information, navigate to the [msedgedriver container on Docker Hub][Doc
 
 Even though Microsoft Edge supports IE Mode, you can't use Microsoft Edge Driver with Microsoft Edge to test sites in IE Mode.  To test sites that require Internet Explorer, use [Internet Explorer Driver][GithubSeleniumHqWikiIEDriver] with Internet Explorer.
 
+## Application Guard
+
+Untrusted sites that use Microsoft Defender Application Guard cannot be automated or manipulated using Microsoft Edge Driver. Application Guard launches untrusted sites in a container, and this container doesn't expose the remote debugging port that Microsoft Edge Driver needs to communicate with the site.
+
+Your enterprise administrator defines what are trusted sites, including cloud resources and internal networks. Sites that are not in the trusted sites list are considered untrusted. Microsoft Edge Driver can automate both InPrivate windows, and sites on the trusted sites list. 
```

Comment:
```suggestion
Your enterprise administrator defines what are trusted sites, including cloud resources and internal networks. Sites that are not in the trusted sites list are considered untrusted. Microsoft Edge Driver can automate both InPrivate windows, and sites on the trusted sites list, including trusted sites that use Application Guard.
```
Do we need an explicit positive statement that WebDriver does work with _trusted_ sites that use Application Guard?  Maybe even start the section with that positive statement of what is supported?  @Reezaali @zoherghadyali

Reply:
SGTM